### PR TITLE
Branch for hw dao7

### PR DESCRIPTION
**Pull request by AntonPanchishko**

### CODE REVIEW

File: src/main/java/taxi/dao/ManufacturerDao.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import java.util.Optional;
 import taxi.model.Manufacturer;
 
-public interface ManufacturerDao {
+public interface ManufacturerDao extends GenericDao<Manufacturer, Long> {
```
Comment (vicire) [inline src/main/java/taxi/dao/ManufacturerDao.java:5]: You can delete all methods under, that you declared in GenericDao. Same in other Dao and Service

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>CRUD</title>
+</head>
+<body>
+<h1>Hello world!</h1>
+</body>
+</html>
```
Comment (vicire) [inline src/main/webapp/WEB-INF/views/index.jsp]: add the empty line in the end

File: src/main/webapp/web.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+         version="4.0">
+    <servlet>
+        <servlet-name>index</servlet-name>
+        <servlet-class>taxi.controllers.IndexController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+</web-app>
```
Comment (vicire) [inline src/main/webapp/web.xml]: add empty line here

**Review by vicire — CHANGES_REQUESTED** [review]
nice, I have only some little remarks :)

**Review by vicire — APPROVED** [review]

**Review by boroda4436 — CHANGES_REQUESTED** [review]
https://mate-academy.github.io/jv-program-fulltime/02_jdbc/checklist/07_servlets_intro_checklist.html

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**